src/renderer/domains/collectives/referendum/resource.ts

```diff
@@ -49,6 +49,18 @@ async function parseProposal(proposal: FrameSupportPreimagesBounded, api: ApiPro
       }
     }
 
+    if (struct.method === 'send') {
+      return {
+        type: 'Whitelist',
+      };
+    }
+
+    if (struct.method === 'spendLocal' || struct.method === 'spend') {
+      return {
+        type: 'Spend',
```

Comment:
it will be good to also parse amount

Reply:
In order to make this feature usefull we actually need to parse both token and amount.
Parsing token is the most complex task here, it works similar to determining a chain. I can elaborate on the details separately